exporter/signalfxexporter/translation/constants.go

```diff
@@ -602,6 +602,26 @@ translation_rules:
     memory.utilization: 100
     cpu.utilization: 100
 
+# Virtual memory metrics
```

Comment:
While you're at it, can you also add rules for the non-default metrics that we need to translate?

Reply:
Added!